xcodeproj/xcodeproj.bzl

```diff
@@ -218,19 +226,19 @@ def make_xcodeproj_rule(*, transition = None):
             allow_empty = False,
             aspects = [xcodeproj_aspect],
         ),
-        "_generator": attr.label(
-            cfg = "exec",
-            # TODO: Use universal generator when done debugging
-            default = Label("//tools/generator:generator"),
-            executable = True,
-        ),
         "_external_file_marker": attr.label(
             allow_single_file = True,
             # This just has to point to a source file in an external repo. It is
             # only used by a local action, so it doesn't matter what it points
             # to.
             default = "@build_bazel_rules_apple//:LICENSE",
         ),
+        "_generator": attr.label(
+            cfg = "exec",
+            # TODO: Use universal generator when done debugging
+            default = Label("//tools/generator:generator"),
+            executable = True,
+        ),
```

Comment:
Small nit: can you submit these formatting changes as a separate PR?

Reply:
It wasn't intentional. My editor runs Buildifier on files that I touch. Let me see what I can do.

Reply:
Done.